gaia-header.js

```diff
@@ -278,9 +278,9 @@ module.exports = component.register('gaia-header', {
   setTitleStyle: function(el, style) {
     debug('set title style', style);
     this.observerStop();
-    el.style.MozMarginStart = style.marginStart + 'px';
-    el.style.MozPaddingStart = style.padding.start + 'px';
-    el.style.MozPaddingEnd = style.padding.end + 'px';
+    el.style.marginLeft = style.marginStart + 'px';
+    el.style.paddingLeft = style.padding.start + 'px';
+    el.style.paddingRight = style.padding.end + 'px';
```

Comment:
Current implementation in Gecko is: we choose left/right depending on the directionality of the element itself.
new spec says: we choose left/right depending on the directionality of the element's parent. (only for margin; for padding it's still the directionality of the element).

Jim's argument was that margin should depend on the directionality of the document, not the directionality of the element. And I think it's the same for us here, actually. If the locale is RTL but the title is in LTR for some reason, the margin should still be decided for RTL.